### ISSUE TITLE
Rebase onto current version of Bootstrap

### ISSUE THREAD

**Issue by jrchamp** [ncstate/ncstate-bootstrap#8]
It looks like this was originally based on Bootstrap v3.1.0

> v3.1.0
> 
> mdo released this on Jan 30, 2014 · 3525 commits to master since this release

There are color contrast issues for the alert classes that are causing accessibility checks to fail. Rather than try to address these items individually, it might be beneficial to benefit from the large number of fixes that have been made to the Bootstrap codebase in the past two and a half years.

If you would like, we could work together on this.

**Comment by bpdeconi** [comment]
This is something I would also be very interested in helping with.

**Comment by apmatthews** [comment]
Thanks for the suggestion and the offers to help.

We have recently been discussing the next steps for NC State Bootstrap. There are several factors we are considering such as what design updates we need to make, how much effort will be involved in adapting our customizations to a new version, how we will make merging core updates easier, what urgent issues there are with the current version, and the upcoming release of Bootstrap 4.

We will post an update here once we have developed a plan for moving forward. In the meantime, if you want to post issues for the most pressing needs, that will help us plan and prioritize.

**Comment by jimjones26** [comment]
Any news on moving to bootstrap 4?

**Comment by jrchamp** [comment]
They have a Bootstrap 4 version, but it's in NC State's GitHub.

**Comment by jimjones26** [comment]
I looked around but did not see anything obvious. Maybe this is the place?

https://github.com/ncstate/utility-bar/blob/master/src/main.scss

**Comment by jrchamp** [comment]
https://brand.ncsu.edu/bootstrap/v4/ takes you to the documentation pages
built with the modified version bootstrap, but the code itself is within
https://github.ncsu.edu/

I guess it would be good to mark this repository as done/archived if it's
not going to be maintained or pull in the new one from upstream.
